tests/unit_tests/test_wave_01.py

```diff
@@ -40,16 +40,17 @@ def test_removing_from_inventory_returns_item():
     assert item not in vendor.inventory
     assert result == item
 
-@pytest.mark.skip
+# @pytest.mark.skip
 def test_removing_not_found_is_false():
     item = "item to remove"
     vendor = Vendor(
         inventory=["a", "b", "c"]
     )
 
     result = vendor.remove(item)
+    assert result == False
```

Comment:
Great start! It would also be a good idea to make sure the inventory length is still 3 and that nothing was accidentally removed.